src/ol/source/Cluster.js

```diff
@@ -63,7 +63,7 @@ import {getUid} from '../util.js';
  * considered for clustering, a custom `geometryFunction` can be defined.
  *
  * If the instance is disposed without also disposing the underlying
- * source `setSource(null)` has to be called to remove the listener reference
+ * source `setSource(undefined)` has to be called to remove the listener reference
```

Comment:
Did `setSource(null)` work previously, i.e. did it remove the source? If so, then this is a breaking change.

Reply:
Yes, passing null removed the source, I changed it again to accept null